cmd/flux/cmd/test_test.go

```diff
@@ -176,9 +180,24 @@ func testGatherFromArchive(t *testing.T, fn gatherFunc, filename string) {
 		}
 	}
 
+	if want, got := 1, len(modules); want != got {
+		t.Errorf("unexpected number of modules -want/+got:\n\t- %d\n\t+ %d", want, got)
+	} else {
+		mod, ok := modules["math"]
+		if !ok {
+			t.Error("missing module \"math\"")
+		} else {
+			f, err := mod.Open("a/a_test.flux")
+			if err != nil {
+				t.Errorf("unexpected error: %s", err)
+			}
+			_ = f.Close()
+		}
+	}
+
 	// Check the contents of each of these.
 	ctx := filesystem.Inject(context.Background(), fs)
-	for _, index := range []int{1, 3} {
+	for _, index := range []int{2, 4} {
```

Comment:
Are these magic numbers?

Reply:
Kind of? They're the index for the files to check from the `archiveFiles` array. Instead of repeating the information, I just referenced them by the index. Since I modified that array I had to modify the indexes. I can add a comment for that.